workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/common/components/StatusBadge/StatusBadge.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayIcon from '@clayui/icon';
+
+import {Status} from '../../utils/constants/status';
+
+type StatusClassname = {
+	[key in string]: string;
+};
+
+interface Props {
+	status: string;
+}
+
+const statusClassName: StatusClassname = {
```

Comment:
can we sort this in alphabetical order?

Reply:
@eliasantosf pls do this in your PR

Reply:
Hey @kylembischof, the ordination has already taken place in this [PR](https://github.com/is-solutions-delivery/liferay-portal/pull/5518/files). It's even on the master.